CHANGELOG.md

```diff
@@ -9,6 +9,20 @@
 ### Non-protocol Changes
 **No Changes**
 
+## [2.10.0]
```

Comment:
I've added this section for changelog, since "unreleased" already has content for 2.11 AFAIU.

Let me know if there is a better way to submit changelog entries for the features stabilized here.